addons/website/static/src/xml/website.ace.xml

```diff
@@ -3,15 +3,32 @@
     <t t-name="website.ace_view_editor">
         <div class="oe_ace_view_editor oe_ace_closed">
             <div class="form-inline oe_ace_view_editor_title">
-                <select id="ace-view-list" class="form-control oe_view_list">
+                <div class="btn-group ace_editor_switcher">
+                    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
+                         <span class="type_switcher_base"><i class="fa fa-html5"/> HTML </span> <span class="caret"></span>
+                    </button>
+                      <ul class="dropdown-menu" role="menu">
+                        <li><a href="javaScript:void(0);" class="type_switcher" data-edit-type="html"><i class="fa fa-html5"/> HTML</a></li>
+                        <li><a href="javaScript:void(0);" class="type_switcher" data-edit-type="css"><i class="fa fa-css3"/> Css</a></li>
+                        <li><a href="javaScript:void(0);" class="type_switcher" data-edit-type="less"><strong>{</strong> less <strong>}</strong></a> </li>
```

Comment:
We use mostly href="#" with event.preventDefault() (in your case is in change_type method)